lib/quicksilver-config-client.js

```diff
@@ -51,6 +60,24 @@ class QuicksilverConfigClient {
   getExchangeSettings() {
     return true;
   }
+
+  /**
+   * Check that configuration path (qsConfigPath) value defined in .env as QS_CONFIG_PATH
+   * is a valid path.
+   */
+  configFileFound() {
+    return new Promise((resolve) => {
+      fs.lstat(this.qsConfigPath, (err, stats) => {
+        if (err) {
+          throw new Error(err);
+        }
+        if (!stats.isFile()) {
+          throw new Error(err);
```

Comment:
And this got lost again:

You need to return your error text here. `err` variable isn't getting set just because `this.qsConfigPath` is not file.